nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/CommentUtils.java

```diff
@@ -129,6 +136,24 @@ public static void checkReceivedMail(List<MailMessage> mails, DocumentModel comm
         }
     }
 
+    /**
+     * Checks the received mail, sent when a comment / annotation is created or updated.
+     *
+     * @param mail the mail to compare
+     * @param commentDocModel the document model of the comment, cannot be {@code null}
+     * @param commentedDocModel the document being commented, cannot be {@code null}
+     * @param event the event, cannot be {@code null}
+     * @param commentEventType the type of comment event {@link org.nuxeo.ecm.platform.comment.api.CommentEvents},
+     *            cannot be {@code null}
+     */
+    public static String checkMailContent(MailMessage mail, DocumentModel commentDocModel,
```

Comment:
same thing as `checkNotifications` will be removed, i think this one too  (`checkMailContent`)

Reply:
ah my lovely methods...:'(